### PR TITLE
ADDITION Bug/1432/disable map highlighting on drag or move

### PR DESCRIPTION
**Pull request by RomanenkoVladimir**
# ADDITION: Disable map highlighting on drag or move 

_**Please read the [CONTRIBUTING.md](https://github.com/MaibornWolff/codecharta/blob/main/CONTRIBUTING.md) before opening a PR.**_

addition: #1432

## Description
- The last PR had a small issue with one test after the most recent change, this fixes it so the pipeline is green

### CODE REVIEW

**Comment by RomanenkoVladimir** [comment]
Just a test change for the previous PR, the test was red because building highlighting is handled differently after the last change to preserve the highlight-on-right-click-functionality

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Fixed 🐞
 
+-   Disable building's highlight effect on moving the map #1432
```
Comment (BridgeAR) [inline CHANGELOG.md:20]:
```suggestion
```

I moved the change to the "changed" part :-)

File: visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts [hunk 2]
```diff
@@ -408,17 +408,13 @@ describe("codeMapMouseEventService", () => {
 				event = { button: ClickType.RightClick, clientX: 0, clientY: 1 }
 			})
 
-			it("should $broadcast a building-right-clicked event with data", () => {
+			it("should $broadcast a building-right-clicked event ", () => {
```
Comment (BridgeAR) [inline visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts:411]:
```suggestion
			it("should $broadcast a building-right-clicked event", () => {
```

It is not obvious that the click is indeed a right click in this case. Is it possible to verify that as well?

**Review by BridgeAR — APPROVED** [review]

File: visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts (hunk 2 above)
Comment (RomanenkoVladimir) [reply visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts]: we fire a button:ClickType.RightClick before each test, shouldnt that cover it?

**Pull request merged**